### PR TITLE
Update system requirements, copyright dates, etc.

### PR DESCRIPTION
**Pull request by DavidStaheli**

### CODE REVIEW

**Comment by yacaovsnc** [comment]
LGTM.

File: source/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 1]
```diff
@@ -459,28 +459,23 @@
...
 					</p>
 					<ul xmlns="">
-						<li>Oracle Java™ 1.6+ or IBM Java™ 1.6+ on Microsoft Windows</li>
-						<li>Apple Java™ 1.6+ on Mac OS X</li>
-						<li>Oracle Java™ 1.6+ on Linux or Solaris</li>
-						<li>IBM Java™ 1.6+ on Linux or AIX</li>
-						<li>HP Java™ 1.6+ on HP-UX</li>
+						<li>Oracle Java™ 1.7+ or IBM Java™ 1.7+ on Microsoft Windows</li>
+						<li>Apple Java™ 1.7+ on Mac OS X</li>
+						<li>Oracle Java™ 1.7+ on Linux</li>
```
Comment (AlexRukhlin) [inline source/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: Why not "Oracle Java™ 1.7+ or IBM Java™ 1.7+ on Linux" as above?

File: translations/de/com.microsoft.tfs.client.eclipse.feature/feature.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿Feature-Name=Team Explorer Everywhere
 Feature-Vendor=Microsoft Corporation
-Feature-Description=Tools f\u00fcr die Verwendung mit Microsoft Team Foundation Server und Visual Studio Team Services. Unterst\u00fctzt Team Foundation Server\u00a02010, 2012, 2013, 2015 und Visual Studio Team Services.
-Feature-Copyright=Copyright (c) 2016, Microsoft Corporation und andere Unternehmen. Alle Rechte vorbehalten.
+Feature-Description=Tools f\u00fcr die Verwendung mit Microsoft Team Foundation Server und Visual Studio Team Services. Unterst\u00fctzt Team Foundation Server\u00a02012, 2013, 2015, 2017 und Visual Studio Team Services.
```
Comment (AlexRukhlin) [inline translations/de/com.microsoft.tfs.client.eclipse.feature/feature.properties]: Do we need the 'NO-BREAK SPACE' (U+00A0) character here?

File: translations/de/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 3]
```diff
@@ -559,7 +554,7 @@ <h4 class="subHeading" xmlns=""><b>So installieren Sie den plattformübergreifen
 		<![endif]--></div>
 	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. Alle Rechte vorbehalten.">&copy; 2016 Microsoft. Alle Rechte vorbehalten.</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. Alle Rechte vorbehalten.">&copy; 2016 Microsoft. Alle Rechte vorbehalten.</span>
```
Comment (AlexRukhlin) [inline translations/de/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: 2017 ?

File: translations/fr/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 4]
```diff
@@ -559,7 +554,7 @@ <h4 class="subHeading" xmlns=""><b>Pour installer le client de ligne de commande
 		<![endif]--></div>
 	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. Tous droits réservés.">&copy; 2016 Microsoft. Tous droits réservés.</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. Tous droits réservés.">&copy; 2016 Microsoft. Tous droits réservés.</span>
```
Comment (AlexRukhlin) [inline translations/fr/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: 2017 ?

File: translations/fr/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 5]
```diff
@@ -531,7 +531,7 @@ <h4 class="subHeading" xmlns=""><b>Pour installer les langues supplémentaires d
 			<tool:tip element="languageFilterToolTip" avoidmouse="false"/>
 		<![endif]--></div>	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. Tous droits réservés.">&copy; 2016 Microsoft. Tous droits réservés.</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. Tous droits réservés.">&copy; 2016 Microsoft. Tous droits réservés.</span>
```
Comment (AlexRukhlin) [inline translations/fr/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: 2017 ?

File: translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 6]
```diff
@@ -559,7 +554,7 @@ <h4 class="subHeading" xmlns=""><b>Team Foundation Server 用のクロスプラ
 		<![endif]--></div>
 	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. All rights reserved.">&copy; 2016 Microsoft. All rights reserved.</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. All rights reserved.">&copy; 2016 Microsoft. All rights reserved.</span>
```
Comment (AlexRukhlin) [inline translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: 2017

File: translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 7]
```diff
@@ -531,7 +531,7 @@ <h4 class="subHeading" xmlns=""><b>プラットフォーム間コマンド ラ
 			<tool:tip element="languageFilterToolTip" avoidmouse="false"/>
 		<![endif]--></div>	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. All rights reserved.">&copy; 2016 Microsoft. All rights reserved.</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. All rights reserved.">&copy; 2016 Microsoft. All rights reserved.</span>
```
Comment (AlexRukhlin) [inline translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: 2017

File: translations/ja/com.microsoft.tfs.client.eclipse.ui/about.ini [hunk 8]
```diff
@@ -1,5 +1,5 @@
 # This is really a Java properties file, and as such MUST be ISO 8859-1 
 # (http://help.eclipse.org/help33/index.jsp?topic=/org.eclipse.platform.doc.isv/reference/misc/about_customization.html)
 
-aboutText=Team Explorer Everywhere\n\u30d0\u30fc\u30b8\u30e7\u30f3 14.0.4\n\n Copyright (c) Microsoft. All rights reserved.\nMIT \u30e9\u30a4\u30bb\u30f3\u30b9\u306b\u3088\u308a\u30e9\u30a4\u30bb\u30f3\u30b9\u3055\u308c\u3066\u3044\u307e\u3059\u3002\u30ea\u30dd\u30b8\u30c8\u30ea\u306e\u30eb\u30fc\u30c8\u306b\u3042\u308b License.txt \u3092\u3054\u89a7\u304f\u3060\u3055\u3044\u3002
+aboutText=Team Explorer Everywhere\n\u30d0\u30fc\u30b8\u30e7\u30f3 14.114.0\n\n Copyright (c) Microsoft. All rights reserved.\nMIT \u30e9\u30a4\u30bb\u30f3\u30b9\u306b\u3088\u308a\u30e9\u30a4\u30bb\u30f3\u30b9\u3055\u308c\u3066\u3044\u307e\u3059\u3002\u30ea\u30dd\u30b8\u30c8\u30ea\u306e\u30eb\u30fc\u30c8\u306b\u3042\u308b License.txt \u3092\u3054\u89a7\u304f\u3060\u3055\u3044\u3002
```
Comment (AlexRukhlin) [inline translations/ja/com.microsoft.tfs.client.eclipse.ui/about.ini]: 115

File: translations/pt/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 9]
```diff
@@ -558,7 +553,7 @@ <h4 class="subHeading" xmlns=""><b>Para instalar o cliente de linha de comando d
 		<![endif]--></div>
 	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. Todos os direitos reservados.">&copy; 2016 Microsoft. Todos os direitos reservados.</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. Todos os direitos reservados.">&copy; 2016 Microsoft. Todos os direitos reservados.</span>
```
Comment (AlexRukhlin) [inline translations/pt/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: 2017

File: translations/pt/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 10]
```diff
@@ -531,7 +531,7 @@ <h4 class="subHeading" xmlns=""><b>Para instalar os idiomas adicionais do client
 			<tool:tip element="languageFilterToolTip" avoidmouse="false"/>
 		<![endif]--></div>	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. Todos os direitos reservados.">&copy; 2016 Microsoft. Todos os direitos reservados.</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. Todos os direitos reservados.">&copy; 2016 Microsoft. Todos os direitos reservados.</span>
```
Comment (AlexRukhlin) [inline translations/pt/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: 2017

File: translations/pt/com.microsoft.tfs.client.eclipse.ui/about.ini [hunk 11]
```diff
@@ -1,5 +1,5 @@
 # This is really a Java properties file, and as such MUST be ISO 8859-1 
 # (http://help.eclipse.org/help33/index.jsp?topic=/org.eclipse.platform.doc.isv/reference/misc/about_customization.html)
 
-aboutText=Team Explorer Everywhere\nVers\u00e3o 14.0.4\n\n Copyright (c) Microsoft. Todos os direitos reservados.\nLicenciado sob a licen\u00e7a MIT. Consulte License.txt na raiz do reposit\u00f3rio.
+aboutText=Team Explorer Everywhere\nVers\u00e3o 14.114.0\n\n Copyright (c) Microsoft. Todos os direitos reservados.\nLicenciado sob a licen\u00e7a MIT. Consulte License.txt na raiz do reposit\u00f3rio.
```
Comment (AlexRukhlin) [inline translations/pt/com.microsoft.tfs.client.eclipse.ui/about.ini]: 115

File: translations/zh_CN/com.microsoft.tfs.client.eclipse.feature/feature.properties [hunk 12]
```diff
@@ -1,5 +1,5 @@
 ﻿Feature-Name=Team Explorer Everywhere
 Feature-Vendor=Microsoft Corporation
-Feature-Description=\u7528\u4e8e Microsoft Team Foundation Server \u548c Visual Studio Team Services \u7684\u5de5\u5177\u3002\u652f\u6301 Team Foundation Server 2010\u3001Team Foundation Server 2012\u3001Team Foundation Server 2013\u3001Team Foundation Server 2015 \u548c Visual Studio Team Services\u3002
-Feature-Copyright=\u7248\u6743\u6240\u6709 (c) 2016\uff0cMicrosoft Corporation \u53ca\u5176\u4ed6\u516c\u53f8\u3002\u4fdd\u7559\u6240\u6709\u6743\u5229\u3002
+Feature-Description=\u7528\u4e8e Microsoft Team Foundation Server \u548c Visual Studio Team Services \u7684\u5de5\u5177\u3002\u652f\u6301 Team Foundation Server 2012\u3001Team Foundation Server 2012\u3001Team Foundation Server 2013\u3001Team Foundation Server 2015 \u548c Visual Studio Team Services\u3002
```
Comment (AlexRukhlin) [inline translations/zh_CN/com.microsoft.tfs.client.eclipse.feature/feature.properties]: 2012 is repeated twice

File: translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 13]
```diff
@@ -458,38 +458,33 @@
...
-						<li>Solaris 8 ～ 11 (SPARC、x64)</li>
-						<li>AIX 5.2 ～ 7.1 (32 および 64 ビット)</li>
-						<li>HP-UX 11i v1 ～ v3 (PA-RISC、Itanium)</li>
 					</ul>
 					<p xmlns="">
 						<b>統合開発環境 (IDE) の要件:</b>
 					</p>
 					<ul xmlns="">
-						<li>Eclipse 3.5 ～ 4.5 に基づく IDE (Windows、Linux、Mac OS X、Solaris、AIX、または HP-UX 上)</li>
+						<li>Eclipse 4.2+ に基づく IDE (Linux、Mac OS X、または Windows 上)</li>
```
Comment (AlexRukhlin) [inline translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: Are you sure of this translation? 
に基づく- means "based on"
または -means "or"
上 - means "up".

File: translations/de/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 14]
```diff
@@ -458,38 +458,33 @@
...
-						<li>Solaris 8 bis 11 (SPARC,x64)</li>
-						<li>AIX 5.2 bis 7.1(32 und 64 Bit)</li>
-						<li>HP-UX 11i v1 bis v3 (PA-RISC, Itanium)</li>
 					</ul>
 					<p xmlns="">
 						<b>Anforderungen an die integrierte Entwicklungsumgebung (IDE):</b>
 					</p>
 					<ul xmlns="">
-						<li>IDEs auf Grundlage von Eclipse 3.5 bis 4.5 unter Windows, Linux, Mac OS X, Solaris, AIX oder HP-UX.</li>
+						<li>IDEs auf Grundlage von Eclipse 4.2+ unter Linux, Mac OS X, oder Windows.</li>
```
Comment (AlexRukhlin) [inline translations/de/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: IDEs - ?

File: translations/zh_CN/com.microsoft.tfs.client.eclipse.feature/feature.properties (hunk 12 above)
Comment (AlexRukhlin) [inline translations/zh_CN/com.microsoft.tfs.client.eclipse.feature/feature.properties]: 2012 is repeated twice while 2017 is missing

File: translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 15]
```diff
@@ -559,7 +554,7 @@ <h4 class="subHeading" xmlns=""><b>安装适用于 Team Foundation Server 的跨
 		<![endif]--></div>
 	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft。保留所有权利。">&copy; 2016 Microsoft。 保留所有权利。</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft。保留所有权利。">&copy; 2016 Microsoft。 保留所有权利。</span>
```
Comment (AlexRukhlin) [inline translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: 2017

File: translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 16]
```diff
@@ -531,7 +531,7 @@ <h4 class="subHeading" xmlns=""><b>安装跨平台命令行客户端附加语言
 			<tool:tip element="languageFilterToolTip" avoidmouse="false"/>
 		<![endif]--></div>	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft。保留所有权利。">&copy; 2016 Microsoft。 保留所有权利。</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft。保留所有权利。">&copy; 2016 Microsoft。 保留所有权利。</span>
```
Comment (AlexRukhlin) [inline translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: 2017

File: translations/zh_TW/com.microsoft.tfs.client.eclipse.feature/feature.properties [hunk 17]
```diff
@@ -1,5 +1,5 @@
 ﻿Feature-Name=Team Explorer Everywhere
 Feature-Vendor=Microsoft Corporation
-Feature-Description=\u8207 Microsoft Team Foundation Server \u4ee5\u53ca Visual Studio Team Services \u642d\u914d\u4f7f\u7528\u7684\u5de5\u5177\u3002\u652f\u63f4 Team Foundation Server 2010\u30012012\u30012013\u30012015 \u548c Visual Studio Team Services\u3002
-Feature-Copyright=Copyright (c) 2016, Microsoft Corporation \u8207\u5176\u4ed6\u5408\u4f5c\u5925\u4f34\u3002\u8457\u4f5c\u6b0a\u6240\u6709\uff0c\u4e26\u4fdd\u7559\u4e00\u5207\u6b0a\u5229\u3002
+Feature-Description=\u8207 Microsoft Team Foundation Server \u4ee5\u53ca Visual Studio Team Services \u642d\u914d\u4f7f\u7528\u7684\u5de5\u5177\u3002\u652f\u63f4 Team Foundation Server 2012\u30012012\u30012013\u30012015 \u548c Visual Studio Team Services\u3002
```
Comment (AlexRukhlin) [inline translations/zh_TW/com.microsoft.tfs.client.eclipse.feature/feature.properties]: 2012 is repeated twice while 2017 is missing

File: translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 18]
```diff
@@ -559,7 +554,7 @@ <h4 class="subHeading" xmlns=""><b>安裝 Team Foundation Server 跨平台命令
 		<![endif]--></div>
 	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. 著作權所有，並保留一切權利。">&copy; 2016 Microsoft. 著作權所有，並保留一切權利。</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. 著作權所有，並保留一切權利。">&copy; 2016 Microsoft. 著作權所有，並保留一切權利。</span>
```
Comment (AlexRukhlin) [inline translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: 2017

File: translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 19]
```diff
@@ -531,7 +531,7 @@ <h4 class="subHeading" xmlns=""><b>安裝 Team Foundation Server 跨平台命令
 			<tool:tip element="languageFilterToolTip" avoidmouse="false"/>
 		<![endif]--></div>	<div id="footer"><hr /><p />
 		<div class="FooterLinks">
-			<span class="FooterCopyright" title="© 2016 Microsoft. 著作權所有，並保留一切權利。">&copy; 2016 Microsoft. 著作權所有，並保留一切權利。</span>
+			<span class="FooterCopyright" title="© 2017 Microsoft. 著作權所有，並保留一切權利。">&copy; 2016 Microsoft. 著作權所有，並保留一切權利。</span>
```
Comment (AlexRukhlin) [inline translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: 2017

File: translations/zh_TW/com.microsoft.tfs.client.eclipse.ui/about.ini [hunk 20]
```diff
@@ -1,5 +1,5 @@
 # This is really a Java properties file, and as such MUST be ISO 8859-1 
 # (http://help.eclipse.org/help33/index.jsp?topic=/org.eclipse.platform.doc.isv/reference/misc/about_customization.html)
 
-aboutText=Team Explorer Everywhere\nVersion 14.0.4\n\n Copyright (c) Microsoft. \u8457\u4f5c\u6b0a\u6240\u6709\uff0c\u4e26\u4fdd\u7559\u4e00\u5207\u6b0a\u5229\u3002\n\u5df2\u7372 MIT License \u6388\u6b0a\u3002 \u8acb\u53c3\u95b1\u5b58\u653e\u5eab\u6839\u76ee\u9304\u4e2d\u7684 License.txt\u3002
+aboutText=Team Explorer Everywhere\nVersion 14.114.0\n\n Copyright (c) Microsoft. \u8457\u4f5c\u6b0a\u6240\u6709\uff0c\u4e26\u4fdd\u7559\u4e00\u5207\u6b0a\u5229\u3002\n\u5df2\u7372 MIT License \u6388\u6b0a\u3002 \u8acb\u53c3\u95b1\u5b58\u653e\u5eab\u6839\u76ee\u9304\u4e2d\u7684 License.txt\u3002
```
Comment (AlexRukhlin) [inline translations/zh_TW/com.microsoft.tfs.client.eclipse.ui/about.ini]: 115

**Comment by paultbarham** [comment]
Changes look good to me - Alex has caught good issues.  I'm OK leaving version at 14.114.0 -- up to you David.

File: translations/de/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm (hunk 14 above)
Comment (DavidStaheli) [inline translations/de/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: @AlexRukhlin I'm not sure what should change related to "IDEs".

**Pull request merged**